subprojects/GOPATH/src/github.com/microsoft/hivedscheduler/pkg/algorithm/cell.go

```diff
@@ -121,30 +118,58 @@ func (c *GenericCell) IncreaseUsedGpuNumAtPriority(p CellPriority, delta int32)
 // PhysicalCell defines a cell in the physical cluster.
 type PhysicalCell struct {
 	GenericCell
-	nodes               []string           // node names inside the cell
-	gpuIndices          []int32            // [-1] for cells at levels higher than node
-	affinityGroup       *AlgoAffinityGroup // affinity group using this cell
-	virtualCell         *VirtualCell       // points to the bound virtual cell
-	preBoundVirtualCell *VirtualCell       // points to the temporarily bound virtual cell (before the binding is confirmed)
-	split               bool               // true when the cell has been split
-	reserved            bool               // true when this is a reserved cell
+	nodes               []string                // node names inside the cell
+	gpuIndices          []int32                 // [-1] for cells at levels higher than node
+	affinityGroup       *AlgoAffinityGroup      // affinity group using this cell
+	virtualCell         *VirtualCell            // points to the bound virtual cell
+	preBoundVirtualCell *VirtualCell            // points to the temporarily bound virtual cell (before the binding is confirmed)
+	split               bool                    // true when the cell has been split
+	reserved            bool                    // true when this is a reserved cell
+	status              *api.PhysicalCellStatus // json representation of the status of this cell to expose to users
```

Comment:
>json representation of the status of this cell to expose to users [](start = 48, length = 65)

Just the "the exposed status of this cell"? 
It is not related to json, it is just golang obj, and can be converted to yaml in future.

Reply:
Similar to [https://github.com/kubernetes/kubernetes/blob/1afb554222605d4d67a911c9bbb0fa5b1049c2dc/pkg/kubelet/kubelet_pods.go#L1409](https://github.com/kubernetes/kubernetes/blob/1afb554222605d4d67a911c9bbb0fa5b1049c2dc/pkg/kubelet/kubelet_pods.go#L1409)

---
In reply to: [388119580](https://github.com/microsoft/pai/pull/4153#discussion_r388119580) [](ancestors = 388119580)